### PR TITLE
refactor(macros): optimize code structure of shouldUpdate

### PR DESCRIPTION
**Pull request by Enside**
Just optimize code structure of shouldUpdate.

### CODE REVIEW

**Comment by jourdain** [comment]
LGTM

File: Sources/macros.js [hunk 1]
```diff
@@ -763,22 +755,13 @@ export function algo(publicAPI, model, numberOfInputs, numberOfOutputs) {
     count = model.numberOfInputs;
     while (count--) {
       if (
-        model.inputConnection[count] &&
-        model.inputConnection[count].filter.shouldUpdate()
+        model.inputConnection[count]?.filter.shouldUpdate() ||
```
Comment (finetjul) [inline Sources/macros.js:758]: @floryst @jourdain are we OK with optional chaining in VTK.JS ?

Comment (jourdain) [reply Sources/macros.js:758]: I thought we are based on the latest update of the toolchain. Am I correct @floryst ?

Comment (floryst) [reply Sources/macros.js:758]: Yep, we handle optional chaining (and null coalescing, for that matter).

Comment (finetjul) [reply Sources/macros.js:758]: Good to know, thanks !

**Pull request merged**